core/src/main/java/io/questdb/network/AbstractIODispatcher.java

```diff
@@ -263,6 +263,8 @@ protected void doDisconnect(C context, int src) {
         if (context == null || context.invalid()) {
             return;
         }
+        context.dumpBuffer();
```

Comment:
is this debug leftover ?

Reply:
no, that is intentional.
idea is that if anything left in the buffer it will be dumped to the logs so we will see that something was not processed.

Reply:
in this case it should be printer as binary rather than assumed to be UTF8 encoded anything. We do have a method for it:

```
Chars.toSink(BinarySequence sequence, CharSink sink)
```

Reply:
changed it